CHANGES/620.feature

```diff
@@ -0,0 +1 @@
+Added support for the 'repositories/reclaim_space' request to be called via the CLI.
```

Comment:
This is user facing, so don't talk about the api, but about features.
Also a changelog for glue is missing.